apidoc/Titanium/UI/RefreshControl.yml

```diff
@@ -55,6 +55,14 @@ properties:
     platforms: [android, iphone, ipad, macos]
     since: { android: "6.2.0", iphone: "3.2.0", ipad: "3.2.0" }
 
+  - name: backgroundColor
+    summary: The background color for the refresh control, as a color name or hex triplet.
+    description: |
+        For information about color values, see the "Colors" section of <Titanium.UI>.
+    type: [String, Titanium.UI.Color]
+    platforms: [iphone, ipad, macos]
+    since: { iphone: "12.4.0", ipad: "12.4.0" }
```

Comment:
Not sure what the latest state was, but what happens if you don't add `macos` here?

Reply:
Sorry, just saw this comment! 

It is there but has `9.2.0`:
![Screenshot_20240504_213506](https://github.com/tidev/titanium-sdk/assets/4334997/716899ce-24c4-48a6-9ffd-8062e5c76f36)
I've fixed it